v2/request.go

```diff
@@ -30,6 +30,20 @@ import (
 
 // Request is the low level request representation.
 type Request struct {
+	// Peer, if present, is the identifier of the task making an inbound
+	// request or the destination identifier for an outbound request.
+	//
+	// The use of the Peer identifier is an implementation detail of each
+	// transport protocol, but generally, if an outbound has a Chooser,
+	// the Peer is a hint to the peer chooser to prefer the given peer if it is
+	// immediately available.
+	// Also generally, if the outbound does not have a Chooser, it should have
```

Comment:
I have a habit of starting each new sentence on a new line so that changes to one sentence do not cause diff churn in subsequent sentences. I can’t find the reference, but this is in an evidently a long forgotten style guide, from the early days of C.